folly/io/async/AsyncServerSocket.cpp

```diff
@@ -727,7 +727,7 @@ void AsyncServerSocket::handlerReady(
     std::chrono::time_point<std::chrono::steady_clock> nowMs =
       std::chrono::steady_clock::now();
     int64_t timeSinceLastAccept = std::max(
```

Comment:
Why not

```cpp
    int64_t timeSinceLastAccept = std::max<int64_t>(
       0,
       nowMs.time_since_epoch().count() -
       lastAccepTimestamp_.time_since_epoch().count());
```

Especially since we're already assigning this to an `int64_t` result? It seems a lot shorter and cleaner than this large `decltype`.